orthoinference/runOrthoinference.sh

```diff
@@ -7,15 +7,17 @@ git pull
 ## Create new jar file with orthoinference code
 mvn clean compile assembly:single
 
+## set args - all required
+configPath="src/main/resources/config.properties"	# Path to main configuration file
+refSpecies="orig"	# Reference species (4-digit abbv)
+projSpecies=(prj1 prj2 prj3)	# Projected species (4-digit abbv); may contain multiple species, space-delimited
+threshold="1"		# Minimum percentage of orthologous catalysts required to project event
```

Comment:
This should be a variable we can modify by calling the script or that is retrieved from the config files

Reply:
Which variable are you referring to?

Reply:
threshold, got it.

Reply:
moving to config file...

Reply:
done